frontend/javascripts/test/sagas/volumetracing/bucket_eviction_helper.ts

```diff
@@ -1,45 +1,67 @@
 import _ from "lodash";
-import mockRequire from "mock-require";
 import "test/sagas/saga_integration.mock";
-import { createBucketResponseFunction } from "test/helpers/apiHelpers";
+import { createBucketResponseFunction, type WebknossosTestContext } from "test/helpers/apiHelpers";
 import Store from "oxalis/store";
 import { OrthoViews, AnnotationToolEnum } from "oxalis/constants";
 import { updateUserSettingAction } from "oxalis/model/actions/settings_actions";
-const { setToolAction } = mockRequire.reRequire("oxalis/model/actions/ui_actions");
-const { setPositionAction } = mockRequire.reRequire("oxalis/model/actions/flycam_actions");
-const { setActiveCellAction, addToLayerAction, startEditingAction, finishEditingAction } =
-  mockRequire.reRequire("oxalis/model/actions/volumetracing_actions");
-export async function testLabelingManyBuckets(t: any, saveInbetween: boolean) {
+import { setToolAction } from "oxalis/model/actions/ui_actions";
+import { setPositionAction } from "oxalis/model/actions/flycam_actions";
+import {
+  setActiveCellAction,
+  addToLayerAction,
+  startEditingAction,
+  finishEditingAction,
+} from "oxalis/model/actions/volumetracing_actions";
+import type { Vector3 } from "oxalis/constants";
+import type Model from "oxalis/model";
+import type { ApiInterface } from "oxalis/api/api_latest";
+import { vi } from "vitest";
+
+// These variables are set by WebknossosTestContext
+declare const api: ApiInterface;
+declare const model: typeof Model;
+declare const mocks: {
+  Request: {
+    sendJSONReceiveArraybufferWithHeaders: (
+      url: string,
+      payload: { data: Array<unknown> },
+    ) => Promise<{ buffer: ArrayBuffer; headers: { "missing-buckets": string } }>;
+  };
+};
```

Comment:
is this really necessary? api, model and mocks seem to be shadowed below by the context variable?